image-micro-service/src/server.js

```diff
@@ -0,0 +1,19 @@
+const express = require('express')
+const resize = require('./resize')
+const server = express()
+
+server.get('/', (req, res) => {
```

Comment:
This looks great! Maybe we can merge this into the backend in order to quickly merge the responsive image feature?

Reply:
how would this work with what we have in the backend? 
I'm looking at this as a potential solution for https://github.com/Human-Connection/Human-Connection/issues/1922
but I don't think we would use `server.get('/')`
I couldn't easily figure out what it should be... I think maybe `server.get('/api/')`, oder? @roschaefer